server/src/main/java/org/elasticsearch/script/field/AbstractScriptFieldFactory.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.script.field;
+
+public abstract class AbstractScriptFieldFactory<T> implements Field<T>, ScriptFieldFactory {
```

Comment:
Can you add a comment like
`// Sharing toScriptField implementation`

Otherwise this class seems more structurally important than it is.

Reply:
Added a comment here with a more thorough explanation.